src/Compilers/Core/Portable/Text/ChangedText.cs

```diff
@@ -181,7 +184,7 @@ public override IReadOnlyList<TextChangeRange> GetChangeRanges(SourceText oldTex
 
         private bool IsChangedFrom(SourceText oldText)
         {
-            for (var info = _info; info != null; info = info.Previous)
+            for (ChangeInfo? info = _info; info != null; info = info.Previous)
```

Comment:
what caused this to change?

Reply:
`var` inferred `ChangeInfo`, but `info.Previous` needs to be able to assign `ChangeInfo?` to that variable. It's one of the particularly frustrating characteristics of `var` with the current NRT implementation.